pages/project-registry/integrating-with-the-project-registry.mdx

```diff
@@ -0,0 +1,187 @@
+| members | address[] | Array of member address(es) to remove. | 
+
+#### updateProfileMetadata
+Will overwrite the existing profile metadata. 
+Only the owner can update profile metadata. 
+
+`updateProfileMetadata(profileId, metadata)`
+| Field | Data Type | Notes |
+| --- | --- | --- |
+| profileId | bytes32 | The id of the profile to update. | 
+| metadata | Metadata | Metadata to add to the profile. | 
+
+#### updateProfileName
+Will set a new name for the profile. 
+Only the owner can update profile metadata. 
+This action will create a new anchorId.
+
+`updateProfileName(profileId, name)`
+| Field | Data Type | Notes |
+| --- | --- | --- |
+| profileId | bytes32 | The id of the profile to update. | 
+| name | string | New name for the profile. | 
+
+#### updateProfilePendingOwner
+Will set a new pending owner. The pending owner will need to call 
+`acceptProfileOwnership` in order for the ownership transfer to be complete.
+Only the profile owner can set a pending owner. 
+
+updateProfilePendingOwner(profileId, pendingOwner)
+| Field | Data Type | Notes |
+| --- | --- | --- |
+| profileId | bytes32 | The id of the profile to update. | 
+| pendingOwner | address | The address for the new profile owner. | 
+
+## Reading Project Registry Data Using The Graph
+
+Project registry data is publicly available through The Graph.
+The Allo V2 Subgraph is available for the following chains:
+* Goerli: https://thegraph.com/hosted-service/subgraph/allo-protocol/allo-v2-goerli
+
+Subgraph data can be accessed either manually through the subgraph Playground,
+or programmatically through The Graph API. For more information about how to set
+up these methods, please see the [Querying the
+Graph](https://thegraph.com/docs/en/querying/querying-the-graph/) page in The
+Graph documentation.
+
+### Building Queries
+To see the data currently available in the Allo V2 Subgraph, use the 
+[Graph Playground](https://thegraph.com/hosted-service/subgraph/allo-protocol/allo-v2-goerli) 
+and click on the explorer. You will also be able to build and test
+queries directly in the UI. 
+
+### Query Examples
+<Tabs items={['js', 'solidity']}>
+  <Tab>
+    ```
+    //Allo subgraph query url
+    const url = `https://api.thegraph.com/subgraphs/name/allo-protocol/allo-v2-goerli`
+    //define the query
+    // This query returns the name of 10 most recently created profiles
+    const query = ` {
+      profiles(first: 10, orderBy: createdAt) 
+      {
+        name
+      }
+    }`;
+    
+    //prepare the request body
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ query: userArticle }),
+    };
+
+    //handle the response
+    fetch(url, options)
+    .then(response => response.json())
+    .then(data => {
+      console.log(data);
+    })
+    .catch(error => {
+      console.error(error);
+    });
+
+    ```
+  </Tab>
+
+  <Tab>
```

Comment:
For this example, can you show how to read from `Registry.sol` from a contract? As an example, in `Allo.sol`, we check that the sending address is a member in a profile. The most likely use case here is writing a strategy that wants to check that an address is a member of a profile

Reply:
Updated! Let me know if this is what you were looking for.